README.md

```diff
@@ -50,7 +50,46 @@ You may need to run this command in order to get your `kubectl` command configur
 
 ##### To simplify deployment of containers on GCE you can push the image to `gcr.io`
 
-I'm following these [instructions](http://scottsmerchek.com/2015/07/24/pushing-to-google-container-registry-from-circleci/)
+Add some (below) environmental variables for each individual project/repo via the CircleCI UI. Click on the gear icon for a repo, then "Environmental Variables" under the ​*Tweaks*​ heading in the left-hand sidebar.
+
+First generate a new JSON key in the [developer console](https://console.developers.google.com/project/lexical-cider-93918/apiui/credential/serviceaccount/account-2@lexical-cider-93918.iam.gserviceaccount.com).
```

Comment:
Might be worth putting the steps to navigate here in the UI rather than a deep link to a specific project's page ...

* log into the [developer console](https://console.developers.google.com/)
* Select API Manager from the Gallery (:hamburger: to the left of "Google Developers Console")
* Select Credentials
* Select (or create) your service account.
* Click "Generate new JSON key"